catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalog.java

```diff
@@ -77,4 +80,7 @@ protected Map<String, PropertyEntry<?>> specificPropertyEntries() {
       }
     };
   }
+
+  /** @return The {@link JdbcColumnDefaultValueConverter} to be used by the catalog */
+  protected abstract JdbcColumnDefaultValueConverter createJdbcColumnDefaultValueConverter();
```

Comment:
provide a default implement?

Reply:
`JdbcColumnDefaultValueConverter` already provides a default implementation. We should make sure that developers are aware of this method.

Reply:
@jerryshao What's your suggestion?